test/unit/metadata.js

```diff
@@ -1013,20 +1013,19 @@ describe('metadata', function() {
         path: 'docname'
       }
       const newDoc = createConflictingDoc(doc)
-      const pathRegExp = conflictRegExp(doc.path)
       should(newDoc.path)
         .be.a.String()
-        .and.match(pathRegExp)
+        .and.match(CONFLICT_PATTERN)
     })
     it('should get the correct _id', () => {
       const doc = {
         path: 'docname'
       }
       const newDoc = createConflictingDoc(doc)
-      const pathRegExp = new RegExp(conflictRegExp(doc.path).source, 'i')
+      const idRegExp = new RegExp(CONFLICT_PATTERN.source, 'i')
```

Comment:
- [ ] The refactored unit tests don't check the part (dirname + basename) before the conflict suffix anymore since we dropped the `conflictRegExp()` function. Which means all of them will still pass even when returning only the conflict suffix. But maybe adding some `.and.startWith(...)` assertions would be good enough, without having to reintroduce the old function.